Test/IsLeapYearTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
```

Comment:
Please merge this class with `GetMinuteQuarterTest` and rename it to be the same as your tested class, with suffix `Test`

Reply:
ok, i will merge in one class